lottie/src/main/java/com/airbnb/lottie/LottieAnimationView.java

```diff
@@ -806,6 +809,24 @@ private void clearComposition() {
     lottieDrawable.clearComposition();
   }
 
+  /**
+   * If rendering via software, Android will fail to generate a bitmap if the view is too large. Rather than displaying
+   * nothing, fallback on hardware acceleration which may incur a performance hit. minSdkVersion is 16. If this is ever
```

Comment:
@podarsmarty The minSdk won't be lowered so you can remove this comment but thanks for thinking of it 😄

Reply:
XD removed